src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const titles = [...table.rows[0].cells];
+const tableMain = document.querySelector('tbody');
+const rows = [...tableMain.rows];
+
+function convertToNumber(string) {
+  return Number(string.replace(/[^\d]/g, ''));
+}
+
+table.addEventListener('click', event => {
+  const index = [...titles].indexOf(event.target);
```

Comment:
use .celIndex instead of indexOf
https://learn.javascript.ru/dom-navigation#dom-navigation-tables